### DISCUSSION TITLE
Vulkan?

### DISCUSSION THREAD

**Discussion by OGNetheri** [baffler/Transparent-Twitch-Chat-Overlay#191] [Ideas]
Is it possible to get the app working over Vulkan borderless window? At least on Baldur's Gate 3 with Vulkan and with Borderless Window setting in game the Chat Overlay isn't staying over the game. Luckily there's dx11 option on the game itself too but for better performance in some cases Vulkan + Twitch Chat Overlay would be better option.

**Comment by baffler** [comment]
I'll look into it and see if I can
